qutebrowser/misc/utilcmds.py

```diff
@@ -247,3 +247,36 @@ def log_capacity(capacity: int):
         raise cmdexc.CommandError("Can't set a negative log capacity!")
     else:
         log.ram_handler.change_log_capacity(capacity)
+
+
+@cmdutils.register(debug=True, name='debug-log-level')
+@cmdutils.argument('level', choices=[level.lower() for level in log.LOG_LEVELS])
+def debug_log_level(level: str):
+    """Change the log level for console logging.
+
+    Args:
+       level: log level for console log.
+    """
+    try:
+        if level in [level.lower() for level in log.LOG_LEVELS.keys()]:
+            log.console_handler.setLevel(log.LOG_LEVELS[level.upper()])
+        else:
+            raise ValueError("Invalid argument, {}".format(level))
```

Comment:
This check isn't needed anymore - `cmdutils` will automatically do the check when you add `choices=...`